### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-aws-lambda-python.git`](https://github.com/cisagov/skeleton-aws-lambda-python.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by jmorrowomni — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

File: src/lambda_handler.py [hunk 1]
```diff
@@ -27,12 +27,21 @@
...
 import logging
 import os
+<<<<<<< HEAD
+=======
+from typing import Any, Optional, Union
+
+# Third-Party Libraries
+import cowsay
+import cowsay.characters
+>>>>>>> e8fed7e947c8a5d19ce75381e0137f1e4fbb83a8
```
Comment (dav3r) [inline src/lambda_handler.py]: @jmorrowomni - just flagging that there are several merge conflicts in this PR that must be resolved.

**Review by dav3r — CHANGES_REQUESTED** [review]

**Comment by jsf9k** [comment]
I'm confused what is going on here, since [GitHub Status](https://githubstatus.com) is all green but clearly this PR page isn't updating according to [the Actions that have been run](https://github.com/cisagov/findings-data-import-lambda/actions/runs/5685113467).

File: src/lambda_handler.py (hunk 1 above)
Comment (jsf9k) [reply src/lambda_handler.py]: @dav3r - The merge conflicts should all be fixed now.

**Review by jsf9k — APPROVED** [review]

**Comment by jsf9k** [comment]
> I'm confused what is going on here, since [GitHub Status](https://githubstatus.com) is all green but clearly this PR page isn't updating according to [the Actions that have been run](https://github.com/cisagov/findings-data-import-lambda/actions/runs/5685113467).

I ended up pushing a dummy commit and then removing it, which got _most_ of the GH Actions to run.  I'm still not sure why the CodeQL checks aren't running.

**Comment by mcdonnnj** [comment]
> > I'm confused what is going on here, since [GitHub Status](https://githubstatus.com) is all green but clearly this PR page isn't updating according to [the Actions that have been run](https://github.com/cisagov/findings-data-import-lambda/actions/runs/5685113467).
> 
> I ended up pushing a dummy commit and then removing it, which got _most_ of the GH Actions to run. I'm still not sure why the CodeQL checks aren't running.

@jsf9k The workflow was disabled due to repository inactivity (I re-enabled it before replying).

**Comment by jsf9k** [comment]
> > > I'm confused what is going on here, since [GitHub Status](https://githubstatus.com) is all green but clearly this PR page isn't updating according to [the Actions that have been run](https://github.com/cisagov/findings-data-import-lambda/actions/runs/5685113467).
> > 
> > 
> > I ended up pushing a dummy commit and then removing it, which got _most_ of the GH Actions to run. I'm still not sure why the CodeQL checks aren't running.
> 
> @jsf9k The workflow was disabled due to repository inactivity (I re-enabled it before replying).

Thank you!

File: Dockerfile [hunk 2]
```diff
@@ -31,6 +31,7 @@ FROM amazon/aws-lambda-python:$PY_VERSION as build-stage
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 ###
+LABEL dummy="dummy"
```
Comment (dav3r) [inline Dockerfile:34]: Can this be deleted?
```suggestion
```

**Review by dav3r — APPROVED** [review]
👍 but please see the one item I noted.

File: Dockerfile (hunk 2 above)
Comment (jsf9k) [reply Dockerfile:34]: [It _is_ deleted in the branch](https://github.com/cisagov/findings-data-import-lambda/blob/93ff85ab92439a9dec03a75a65bedc30ceef6b42/Dockerfile#L31-L35), but the PR isn't updating for some reason.

Comment (dav3r) [reply Dockerfile:34]: Ah, I see.  GitHub weirdness strikes again!

**Pull request merged**